phoenix-core/src/it/java/org/apache/phoenix/end2end/CreateTableIT.java

```diff
@@ -621,55 +623,96 @@ public void testMultiTenantImmutableTableMetadata() throws Exception {
         }
     }
 
+    private void verifyUCFValueInSysCat(String tableName, String createTableString, Properties props, long expectedUCFInSysCat) throws SQLException {
+        String readSysCatQuery = "SELECT TABLE_NAME, UPDATE_CACHE_FREQUENCY FROM SYSTEM.CATALOG WHERE "
+                + "TABLE_NAME = '" + tableName + "'  AND TABLE_TYPE='u'";
+
+        Connection connection = DriverManager.getConnection(getUrl(), props);
```

Comment:
nit: Put connection object in a try-with-resources

Reply:
Done.